### PR TITLE
Lesson 3.1: CSS Styled Components SVG

### PR DESCRIPTION
**Pull request by nihalerdal**
Here is my lesson_3_1 pull request for the review.

### CODE REVIEW

File: src/AddTodoForm.module.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  border-color: transparent;
+  border: 0;
+  width: auto;
+  max-width: 90px;
+  background-color: #7984e6;
+  cursor: pointer;
+  padding: 12px;
+  margin-right: 3px;
+  margin-left: 5px;
+  width: 25%;
```
Comment (AnnaPestova1) [inline src/AddTodoForm.module.css:21]: you have width property on the lines 15 and 22 in the .addButton

File: src/AddTodoForm.module.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    flex-direction: column;
+    align-items: center;
+    gap: 1px;
+  }
+}
+
+@media (min-width: 768px) {
+  .form {
+    flex-direction: row;
+    width: 100%;
```
Comment (AnnaPestova1) [inline src/AddTodoForm.module.css]: you already has width for .form: 100% on the line 3. Do you need the same code here?

File: src/AddTodoForm.module.css [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+   .form {
+    flex-direction: column;
+    align-items: center;
+    gap: 1px;
+  }
+}
+
+@media (min-width: 768px) {
+  .form {
+    flex-direction: row;
```
Comment (AnnaPestova1) [inline src/AddTodoForm.module.css]: row is a default flex-direction.

**Review by AnnaPestova1 — APPROVED** [review]
Hi Nihal! I like your design! Great work.

File: src/AddTodoForm.module.css (hunk 2 above)
Comment (nihalerdal) [reply src/AddTodoForm.module.css]: Removed!👍🏻

**Pull request merged**